stdlib/Random/docs/src/index.md

```diff
@@ -285,7 +285,10 @@ The API is not clearly defined yet, but as a rule of thumb:
    should be defined for this specific RNG, if they are needed;
 2) other documented `rand` methods accepting an `AbstractRNG` should work out of the box,
    (provided the methods from 1) what are relied on are implemented),
-   but can of course be specialized for this RNG if there is room for optimization.
+   but can of course be specialized for this RNG if there is room for optimization;
+3) `copy` for pseudo-RNGs should return an independent copy that generates the exact same random sequence as the
+   original from that point when called in the same way. When this is not feasible (eg hardware-based RNGs),
```

Comment:
Isn't the norm to spell "e.g." with 2 dots?

Reply:
Sure! Will update the content.